### DISCUSSION TITLE
Any way to use mermaid collaboratively + concurrent editor

### DISCUSSION THREAD

**Discussion by danyg** [mermaid-js/mermaid#3959] [Q&A]
Hi I would like to use mermaid to generate a team meeting where together we can come up with the design of a feature, in real-time.

Ideally would be some editor that allows us all to connect and edit the document/documents, in real-time a la Google Docs, Miro, Figma, etc...

We use Miro right now. But the miro-mermaid plugin ATM doesn't allow to modify the mermaid-source code so is quite useless.
I've also checked Google Drive and even allows you to change the source code, but the way to do it is kind of convoluted.

I think notion does a way better job at this, but we don't use it at my company. 😞 

Does anyone know about an alternative that could help us with this?
Much appreciated!

**Comment by ooker777** [comment]
How about using Git?

**Reply by danyg** [reply]
@ooker777 thanks for your reply, I was thinking more of a real-time collaborative concept like Google Drive, Miro, Figma, Notion, etc... Realtime collaboration rather than ASYNC collaboration.
Unless there is a way to use git in that way that I'm not aware.

EDIT: I updated the original question so it's clearer.

**Comment by ooker777** [comment]
Well you can auto pull and auto push in both sides?

Vào 15:49, T.5, 5 Th1, 2023 danyg ***@***.***> đã viết:

**Comment by haiodo** [comment]
May be I'm a bit late, but you could try use [Huly](https://github.com/hcengineering/platform) platform. It support mermaid diagrams and could meet your needs.

Please check a guide: [Huly + Mermaid](https://github.com/hcengineering/platform/wiki/Simple-diagrams-in-Huly-with-Mermaid)

**Comment by ooker777** [comment]
Well you can use it in VS Code with the [Live Share](https://visualstudio.microsoft.com/services/live-share/) feature?
